Jakefile.js

```diff
@@ -105,7 +105,7 @@ var harnessCoreSources = [
     "projectsRunner.ts",
     "loggedIO.ts",
     "rwcRunner.ts",
-    "userRunner.ts",
+    "externalCompileRunner.ts",

```

Comment:
Jakefile, but not gulp?

The official TS homepage refers to gulp. Does it mean both gulp/jake need to be used, depending on area of the project?

Reply:
If they don't behave the same it's a bug.

Reply:
Gulp uses the tsconfig edited below.